modules/fx/arrow-fx-rx2/src/test/kotlin/arrow/fx/SingleKTests.kt

```diff
@@ -49,7 +49,7 @@ class SingleKTests : UnitSpec() {
   }
 
   val CS = SingleK.concurrent(object : Dispatchers<ForSingleK> {
-    override fun default(): CoroutineContext = Schedulers.io().asCoroutineDispatcher()
+    override fun default(): CoroutineContext = Schedulers.computation().asCoroutineDispatcher()
```

Comment:
Why?

Reply:
Oh, this can actually be reverted. I remember you told me that you needed `io` for the laws to pass, and I was just curious to see if that was still the case. I had a git fuck up and I actually broke some Rx concurrency operators but that commit is gone :/ This is left over from debugging, shall I just remove the last commit from this PR all together to remove the Rx changes completely?

I am wrote a non-arrow version of the operators with an adapter layer to run the laws. I need to wrap it up and make it public to ask for help with the `UndeliveredException`. ETA: after release and maybe that can warrants a minor release.

Reply:
Yep, whatever's necessary hahah